### DISCUSSION TITLE
Fuzzy find across project files (not git files)

### DISCUSSION THREAD

**Discussion by davidarny** [LazyVim/LazyVim#2988] [Q&A] [answered]
As per [documentation](https://www.lazyvim.org/plugins/editor#telescopenvim-optional):

> Fuzzy finder. The default key bindings to find files will use Telescope's **find_files or git_files depending on whether the directory is a git repo**.

This mean both `<leader>ff` and `<leader>fF` open git files search, so I can't search for gitignored files (i.e. .env files). Is it possible to configure LazyVim so that `<leader>ff` will search across project files including gitignored?

**Comment by dpetka2001** [comment]
Doesn't `<A-i>` when Telescope picker is open do what you want?

**Reply by rdlu** [reply]
I'm interested on this as well, but `meta-i` is "blinking" the telescope screen and it's not displaying my env files.

So I usually open another tmux tab just to open `nvim .env` or `nvim .envrc`.

**Reply by dpetka2001** [reply, marked as answer]
Does `:Telescope find_files no_ignore=true` show your env files?

**Reply by davidarny** [reply]
@dpetka2001 each `:Telescope find_files no_ignore=true` works

**Reply by davidarny** [reply]
I made a remap and now it's ok, thanks! But still weird to me why to have 3 different keybindings all doing same thing (`<leader>ff`, `<leader>fF`, `<leader>fg` in case of git repo)

**Reply by dpetka2001** [reply]
That's what `<A-i>` also does and it works on my laptop (I just press `<A-i>` and it shows the ignored files). Maybe `<A-i>` is not recognized in your terminal emulator?

**Reply by davidarny** [reply]
@dpetka2001 my which key says it's some `textobject_select()` for that keybinding

<img width="827" alt="57187" src="https://github.com/LazyVim/LazyVim/assets/17799810/cc2f9ad8-2c61-4635-ad1f-09b2e75ac28b">

Also I don't find such keybinding anywhere in the [docs](https://www.lazyvim.org/keymaps). Is it telescope's default keybinding or you remap?

**Reply by dpetka2001** [reply]
It's an internal Telescope mapping that Lazyvim provides, thus not being shown in `Telescope keymaps`. If you press `<C-/>` while in insert mode in Telescope prompt (or `?` in normal mode) it will show you the available mappings for the picker.

**Reply by dpetka2001** [reply]
You can find it in the [docs here](https://www.lazyvim.org/plugins/editor#telescopenvim-optional) or [here in the code](https://github.com/LazyVim/LazyVim/blob/bb36f71b77d8e15788a5b62c82a1c9ec7b209e49/lua/lazyvim/plugins/editor.lua#L204-L252).
